scripts/exwave_prdgen_gridded.sh.ecf

```diff
@@ -0,0 +1,315 @@
+# Origination  : 05/02/2007                                                   #
+# Last update  : 05/04/2007                                                   # 
+#                                                                             #
+# May 2020  Roberto.Padilla@noaa.gov, Henrique.HAlves@noaa.gov                # 
+#         - Merging wave scripts to GFSv16 global workflow                    #
+#                                                                             #
+###############################################################################
+# --------------------------------------------------------------------------- #
+# 0.  Preparations
+# 0.a Basic modes of operation
+ set -xa
+ # Use LOUD variable to turn on/off trace.  Defaults to YES (on).
+ export LOUD=${LOUD:-YES}; [[ $LOUD = yes ]] && export LOUD=YES
+ [[ "$LOUD" != YES ]] && set +x
+ cd $DATA
+ postmsg "$jlogfile" "HAS BEGUN on `hostname`"
+ msg="Starting MWW3 GRIDDED PRODUCTS SCRIPT"
+ postmsg "$jlogfile" "$msg"
+# Output grids
+ grids=${grids:-ao_9km at_10m ep_10m wc_10m glo_30m}
+ maxtries=${maxtries:-720}
+# 0.b Date and time stuff
+ export date=$PDY
+ export YMDH=${PDY}${cyc}
+ echo ' '
+ echo '                         ****************************'
+ echo '                         *** MWW3 PRODUCTS SCRIPT ***'
+ echo '                         ****************************'
+ echo "                                       $date $cycle"
+ echo ' '
+ echo "Starting at : `date`"
+ echo ' '
+ echo "   AWIPS grib fields"
+ echo "   Wave  Grids       : $grids"
+ echo ' '
+ [[ "$LOUD" = YES ]] && set -x
+
+# --------------------------------------------------------------------------- #
+# 1.  Get necessary files
+ echo ' '
+ echo 'Preparing input files :'
+ echo '-----------------------'
+ [[ "$LOUD" = YES ]] && set -x
+#=======================================================================
+ 
+ ASWELL=(SWELL1 SWELL2) # Indices of HS from partitions
+ ASWPER=(SWPER1 SWPER2) # Indices of PERIODS from partitions 
+ ASWDIR=(SWDIR1 SWDIR2) # Indices of DIRECTIONS from partitions 
+                                #  (should be same as ASWELL)
+ #export arrpar=(WIND UGRD VGRD HTSGW PERPW DIRPW WVHGT WVPER WVDIR WDIR ${ASWELL[@]} ${ASWDIR[@]} ${ASWPER[@]})
+ export arrpar=(WIND WDIR UGRD VGRD HTSGW PERPW DIRPW WVHGT ${ASWELL[@]} WVPER ${ASWPER[@]} WVDIR ${ASWDIR[@]} )
+ export nparam=`echo ${arrpar[@]} | wc -w`
+ fhcnt=$fstart
+ while [ $fhcnt -le $FHMAX_WAV ]; do
+   fhr=$(printf "%03d" $fhcnt)
+   for grdOut in $grids;do
+     case $grdOut in
+       ao_9km)  grdID='arctic.9km' ;;
+       at_10m)  grdID='atlocn.0p16' ;;
+       ep_10m)  grdID='epacif.0p16' ;;
+       wc_10m)  grdID='wcoast.0p16' ;;
+       glo_30m) grdID='global.0p25' ;;
+       *)       grdID= ;;
+     esac
+     GRIBIN=$COMIN/$RUNwave.$cycle.$grdID.f${fhr}.grib2
+     GRIBIN_chk=$GRIBIN.idx
+
+     icnt=1
+     while [ $icnt -lt 1000 ]
```

Comment:
Most of the code puts the control structure opener on the same line (while; do, if; then, etc.), but in a few places they are broken up like here. The style should at least be consistent within the same file.